public/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+define("APP_ROOT", dirname(__DIR__));
+define('DS', DIRECTORY_SEPARATOR);
+
+// Autolaod class
+spl_autoload_register(function ($class) {
+  $file = str_replace('\\', DIRECTORY_SEPARATOR, $class) . '.php';
+  $file = APP_ROOT . DIRECTORY_SEPARATOR . $file;
+  if (file_exists($file)) {
+    require $file;
+    return true;
+  }
+  return false;
+});
+
+
+use App\Functions\Router;
+use App\Functions\Helpers;
```

Comment:
Autoloader sa síce spustí, no, naloadovaná classa - kt. re requestnutá v PSR-4 formáte `App\Functions\Router` (čo je v poriadku) - nekoreluje so štruktúrou súborov (reálne je App folder `app` a podobne aj `functions`). Tebe to išlo keď si to spustil ?

Reply:
Áno išlo mi to, neviem ako to rieši veľké/malé písmená